From 571d23cf91e5906ae61eae1201de816ed30f582e Mon Sep 17 00:00:00 2001
From: Yuxi Sun <b36102@freescale.com>
Date: Tue, 31 May 2011 09:42:05 +0800
Subject: [PATCH 985/998] ENGR00144088 MX5 clock: Fix ssi_ext1_clk is always on without alsa playing

Set ssi_ext1_clk default disabled when board boot up.

Signed-off-by: Yuxi Sun <b36102@freescale.com>
---
 arch/arm/mach-mx5/clock.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx5/clock.c b/arch/arm/mach-mx5/clock.c
index c051fd8..1a6d5aa 100644
--- a/arch/arm/mach-mx5/clock.c
+++ b/arch/arm/mach-mx5/clock.c
@@ -4988,7 +4988,6 @@ int __init mx53_clocks_init(unsigned long ckil, unsigned long osc, unsigned long
 	clk_disable(&ssi_ext1_clk);
 	clk_set_parent(&ssi_ext1_clk, &pll2_sw_clk);
 	clk_set_rate(&ssi_ext1_clk, 24000000);
-	clk_enable(&ssi_ext1_clk);
 	clk_set_parent(&ssi_ext2_clk, &ssi2_clk[0]);
 
 	/* move usb_phy_clk to 24MHz */
-- 
1.7.4.1

